src/programs/project3d/project3d.cpp

```diff
@@ -129,7 +147,29 @@ bool Project3DApp::DoCalculation()
 	cisTEMParameterLine input_parameters;
 
 	cisTEMParameters input_star_file;
-	input_star_file.ReadFromcisTEMStarFile(input_star_filename);
+	if (project_based_on_star == true) input_star_file.ReadFromcisTEMStarFile(input_star_filename);
+
+	EulerSearch	global_euler_search;
+	ParameterMap parameter_map; // needed for euler search init
+	parameter_map.SetAllTrue();
+
+	if (project_based_on_star == false)
+	{
+
+		global_euler_search.InitGrid(my_symmetry, angular_step, 0.0f, 0.0f, 360, 180, 180, pixel_size / 8, parameter_map, 20);
```

Comment:
Actually, it looks to me like this will not return the same grid every time, because `InitGrid` calls `CalculateGridSearchPositions` with `random_start_angle = true` so that we will get different projections every time. Is this what we want out of this program?

Reply:
CalculateGridSearchPositions is called again later with false..